packages/@react-spectrum/tag/src/Tag.tsx

```diff
@@ -55,7 +54,6 @@ export function Tag<T>(props: SpectrumTagProps<T>) {
       {...tagRowProps}>
       <div
         {...mergeProps(tagProps, hoverProps, focusProps)}
-        role="gridcell"
```

Comment:
where did this go?

Reply:
it's declared here. i did not realize it before https://github.com/adobe/react-spectrum/blob/f4468f6aa73d8aa3317a16407dfa556b7487a726/packages/%40react-aria/tag/src/useTag.ts#L53